deltachat-rpc-client/src/deltachat_rpc_client/rpc.py

```diff
@@ -14,9 +14,7 @@ def __init__(self, accounts_dir: Optional[str] = None, **kwargs):
         if accounts_dir:
             kwargs["env"] = {
                 **kwargs.get("env", os.environ),
-                "DC_ACCOUNTS_PATH": os.path.abspath(
-                    os.path.expanduser(str(accounts_dir))
-                ),
+                "DC_ACCOUNTS_PATH": str(accounts_dir),
```

Comment:
does the account manager expands this to an absolute path? or will deltachat-rpc-server do it?

Reply:
> It will also probably return relative paths for blobs.

not sure what you mean here, i think it is required that `dc_msg_get_file()` and related `dc_get_config()` continue returning absolute paths - however, i also do not see that this is changed by this pr :)